modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -44,44 +44,51 @@ spec:
 
 .Procedure
 
-To use the tool on the command line: 
+To use the tool on the command line:
 
-. Run the following command:
+. From the terminal, run:
++
+----
+podman login registry.redhat.io
+----
++
+
+. Run:
```

Comment:
Usually we try to describe what the following command is doing. So you could say something like "To pull the xyz image, run the following command:"

I'd use the whatever wording/format you choose for the other steps as well for consistency

Reply:
Done